### ISSUE TITLE
MKtl refactor: creating from shortname broken

### ISSUE THREAD

**Issue by LFSaw** [ModalityTeam/Modality-toolkit#206]
```
a = MKtl('nnkn20'); // throws an error 

// but seems to create an MKtl in MKtl.all
MKtl.all;
// -> ( 'nnkn20': MKtl('nnkn20') )

// so that this tries to set things in the existing MKtl:
a = MKtl('nnkn20', "nanoKONTROL2"); //
// posts 
	//** To change the description of an MKtl, do:MKtl.rebuild(<desc>)

// consequently, these are broken from then on
a.elementAt(\kn)



a.elementAt(\kn)
// >> should return MKtElementGroup, no?
```

**Comment by adcxyz** [comment]
last test - MKtElementGroups are back

**Comment by adcxyz** [comment]
closed with commit #414db8ea24ca53af30be66b6c6305e3aec4111ef
 in refactor_test2.
